compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -1183,8 +1183,6 @@ tests Tools{damlc,repl,validate,davlDar,oldProjDar} = testGroup "Data Dependenci
             ]
         callProcessSilent damlc [ "build", "--project-root", tmpDir </> "proj" ]
 
-    -- TODO https://github.com/digital-asset/daml/issues/8020
-    --   Replace with a simpleImportTest once exceptions are stable.
```

Comment:
Should we allow specifying versions in `simpleImportTest`? Afaict that’s the only difference here.

Reply:
Ah I guess we also run `damlc test` so not quite.

Reply:
We probably should refactor these tests though.